esp-hal/src/dma/mod.rs

```diff
@@ -511,7 +526,7 @@ macro_rules! dma_circular_buffers_chunk_size {
 macro_rules! dma_descriptors_chunk_size {
```

Comment:
Now that we have `dma_tx_buffer`, are these chunk macros still useful?

Reply:
Hmm - I was thinking that they would live till both DmaRxBuff supporters param and all of the drivers use the Buf's. Then they are toast.

Reply:
Ah fairs 👍